line_obfuscator.go

```diff
@@ -13,10 +13,6 @@ import (
 )
 
 const (
-	// PosMax is the largest line or column value that can be represented without loss.
-	// Source: https://go.googlesource.com/go/+/refs/heads/master/src/cmd/compile/internal/syntax/pos.go#11
-	PosMax = 1 << 30
-
```

Comment:
there's just one constant left now, so you could remove the parens if you'd like

Reply:
One sec

Reply:
Done